Description: Specify default config in the table of parameters
  Previously, this configuration was shipped as a big config file.
  This changed in the 8.2.x series and now needs to go into the table of
  parameters -- which is not (yet) comprehensive. Hence, a few variables
  still need to be present in the default config file too.
Forwarded: not-needed
Bug-Debian: http://bugs.debian.org/769100
Author: Michael Hanke <mih@debian.org>

diff --git a/src/condor_utils/param_info.in b/src/condor_utils/param_info.in
index b04e410506..9f7f373c5f 100644
--- a/src/condor_utils/param_info.in
+++ b/src/condor_utils/param_info.in
@@ -47,7 +47,7 @@ default=$(LOG)/KernelTuning.log
 type=path
 
 [LINUX_KERNEL_TUNING_SCRIPT]
-default=$(LIBEXEC)/linux_kernel_tuning
+default=/etc/condor/linux_kernel_tuning
 type=path
 
 [COLLECTOR_ENVIRONMENT]
@@ -859,7 +859,7 @@ default=$(RELEASE_DIR)/include
 type=path
 
 [ETC]
-default=$(RELEASE_DIR)/etc
+default=/etc/condor
 type=path
 
 [XAUTHORITY_USERS]
@@ -904,7 +904,7 @@ type=int
 tags=accountant
 
 [SPOOL]
-default=$(LOCAL_DIR)/spool
+default=$(LOCAL_DIR)/spool/condor
 type=path
 tags=schedd,accountant
 
@@ -1422,7 +1422,7 @@ type=string
 tags=daemons
 
 [COLLECTOR_NAME]
-default=My Pool - $(CONDOR_HOST)
+default=Debian Condor Pool - $(CONDOR_HOST)
 type=string
 usage=the UW-Madison Computer Science Condor Pool is ``UW-Madison CS''.
 tags=collector
@@ -1448,7 +1448,7 @@ type=int
 tags=collector
 
 [CONDOR_ADMIN]
-default=root@$(FULL_HOSTNAME)
+default=root@localhost
 type=string
 customization=normal
 description=HTCondor sends email to this address on pool failure
@@ -1774,7 +1774,7 @@ tags=MAIL,
 
 [MAIL]
 # default location for mail on RHEL is /bin/mail, default on debian is /usr/bin/mail
-default=/bin/mail
+default=/usr/bin/mail
 win32_default=$(BIN)\condor_mail.exe
 type=path
 tags=MAIL,
@@ -2280,7 +2280,7 @@ type=bool
 tags=condor_config
 
 [REQUIRE_LOCAL_CONFIG_FILE]
-default=true
+default=false
 win32_default=false
 type=bool
 tags=condor_config
@@ -2607,7 +2607,7 @@ type=string
 tags=starter,StarterHookMgr
 
 [JAVA_BENCHMARK_TIME]
-default=2
+default=0
 type=int
 usage=If this time is zero or undefined, no Java benchmarks will be run.
 tags=starter,java_detect
@@ -4310,7 +4310,7 @@ description=Classad expression used to set virtual memory rlimit for job
 tags=starter
 
 [INTERACTIVE_SUBMIT_FILE]
-default=$(LIBEXEC)/interactive.sub
+default=/etc/condor/interactive.sub
 version=7.9.0
 type=string
 description=Location of default submit file for condor_submit -i
@@ -4480,7 +4480,7 @@ description=Ganglia Library Search Path
 tags=condor_config
 
 [GANGLIA_LIB64_PATH]
-default=/lib64,/usr/lib64,/usr/local/lib64
+default=/lib,/usr/lib,/usr/local/lib
 type=string
 description=Ganglia 64-bit Library Search Path
 tags=condor_config
@@ -5193,7 +5193,7 @@ type=path
 default=9620
 
 [CRED_STORE_DIR]
-default=$(LOCAL_DIR)/cred_dir
+default=$(LOCAL_DIR)/lib/condor/cred_dir
 type=path
 [CREDD_ARGS]
 default=-p $(CREDD_PORT) -f
diff --git a/src/condor_examples/condor_config.generic.debian.patch b/src/condor_examples/condor_config.generic.debian.patch
index 716edbe07c..967e029189 100644
--- a/src/condor_examples/condor_config.generic.debian.patch
+++ b/src/condor_examples/condor_config.generic.debian.patch
@@ -31,7 +31,7 @@
  #LOCAL_CONFIG_DIR_EXCLUDE_REGEXP = ^((\..*)|(.*~)|(#.*)|(.*\.rpmsave)|(.*\.rpmnew))$
  
  ##  Use a host-based security policy. By default CONDOR_HOST and the local machine will be allowed
-@@ -50,5 +48,27 @@
+@@ -50,5 +48,33 @@
  #FLOCK_TO = condor.cs.wisc.edu, cm.example.edu
  
  ##--------------------------------------------------------------------
@@ -47,7 +47,7 @@
 +RUN     = $(LOCAL_DIR)/run/condor
 +LOG     = $(LOCAL_DIR)/log/condor
 +LOCK    = $(LOCAL_DIR)/lock/condor
-+SPOOL   = $(LOCAL_DIR)/lib/condor/spool
++SPOOL   = $(LOCAL_DIR)/spool/condor
 +EXECUTE = $(LOCAL_DIR)/lib/condor/execute
 +BIN     = $(RELEASE_DIR)/bin
 +LIB     = $(RELEASE_DIR)/lib/condor
@@ -60,3 +60,9 @@
 +PROCD_ADDRESS = $(RUN)/procd_pipe
 +
 +##  Install the condor-single-node package to run HTCondor on a single node
++
++# Don't phone home
++CONDOR_DEVELOPERS = NONE
++CONDOR_DEVELOPERS_COLLECTOR = NONE
++
++SSH_TO_JOB_SSHD_CONFIG_TEMPLATE = /etc/condor/condor_ssh_to_job_sshd_config_template
